src/main/java/se/liu/ida/hefquin/cli/modules/ModFederation.java

```diff
@@ -200,6 +203,19 @@ protected void addBRTPFServers( final List<String> uris ) {
 			addBRTPFServer(uri);
 	}
 
+	protected void addSPARQLEndpoint( final String sparqlEndpointValue, final String pathToVocabularyMappings ) {
```

Comment:
Here, we can keep both versions of 'addSPARQLEndpoint' method or combine them into one method and check if the 'pathToVocabularyMappings' parameter is empty. 
@hartig  Which one do you think is better?

Reply:
These should have been merged because there is a lot of code duplication between them. However, if you create the `VocabularyMapping` separately (as suggested in one of my other comments), then this question here becomes irrelevant. Instead, the `addSPARQLEndpoint` function would simply be extended with a second argument for passing the `VocabularyMapping` object (or `null`). And the same would have to be done for the other `add***` functions.